tests/conftest.py

```
@@ -5,14 +5,112 @@
+    # 1. Clean up orphaned python HTTP server sub-processes
+    try:
+        if os.name == "nt":  # Windows
+            subprocess.run(
+                [
+                    "taskkill",
+                    "/F",
+                    "/IM",
+                    "python.exe",
+                    "/FI",
+                    "WINDOWTITLE eq http.server*",
+                ],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)